metron-platform/metron-solr/src/test/java/org/apache/metron/solr/integration/SolrMetaAlertIntegrationTest.java

```diff
@@ -113,12 +114,12 @@ protected String getDefaultSourceTypeField() {
       }
     };
 
-
+    SolrClient solrClient = solrDao.getSolrClient();
     SolrMetaAlertSearchDao searchDao = new SolrMetaAlertSearchDao(
-        solrDao.getSolrClient(solrDao.getZkHosts()),
+        solrClient,
         solrDao.getSolrSearchDao(), config);
-    SolrMetaAlertRetrieveLatestDao retrieveLatestDao = new SolrMetaAlertRetrieveLatestDao(solrDao);
-    SolrMetaAlertUpdateDao updateDao = new SolrMetaAlertUpdateDao(solrDao, searchDao,
+    SolrMetaAlertRetrieveLatestDao retrieveLatestDao = new SolrMetaAlertRetrieveLatestDao(solrClient, solrDao);
```

Comment:
Yeah, I think this is much clearer. Then we don't have to dig through a morass of code just to figure out who is creating what client objects and who is secretly depending on them under the hood.